src/cavendish_particle_tracks/_widget.py

```diff
@@ -404,13 +414,36 @@ def _on_click_load_data(self) -> None:
             self.msg.show()
             return
 
-        for subdir, stack_name in zip(
-            folder_subdirs, ["stack1", "stack2", "stack3"]
-        ):
-            stack = imread(subdir + "/*")
-            self.viewer.add_image(stack, name=stack_name)
-            # TODO: investigate the multiscale otption.
-        self.viewer.dims.axis_labels = ("Event", "Y", "X")
+        def crop(array):
+            # Crops view 1 and 2 to same size as view 3 by removing whitespace
+            # on left, as images align on the right.
+            return array[:, :, -8377:, :]
+
+        stacks = []
+        for subdir in folder_subdirs:
+            stack: da = imread(subdir + "/*")
+            if not self.test_mode:
+                stack = crop(stack)
```

Comment:
?

Reply:
@palvarezc  was this a fix to get the tests to pass?

Reply:
Yes, but I think it is not needed for this PR. I modified the affected error, so that we dont have to cheat it.